docs/HowTo/Use-Privacy/Sign-Privacy-Marker-Transactions.md

```diff
@@ -0,0 +1,14 @@
+description: How to sign a Privacy Marker Transaction
+<!--- END of page meta data -->
+
+# Signing Private Marker Transactions
+
+Private Marker Transactions can be signed with a random key or private key. To sign Private Marker Transactions with a private key, use [`--privacy-marker-transaction-signing-key-file`](../../Reference/Pantheon-CLI/Pantheon-CLI-Syntax.md#privacy-marker-transaction-signing-key-file) when starting Pantheon.
```

Comment:
```suggestion
Private Marker Transactions are signed with a random or specified key. To sign Private Marker Transactions with a private key, use [`--privacy-marker-transaction-signing-key-file`](../../Reference/Pantheon-CLI/Pantheon-CLI-Syntax.md#privacy-marker-transaction-signing-key-file) when starting Pantheon.
```

'are' is more direct that 'can be' 

It probably needs a slight rewording to reflect that if you're signing with a random key, it's a different key for each signing.  You could interpret the current word as the same random key being used over and over for that node. 

'private key' doesn't sound quite right because the random key is also a private key.